### DISCUSSION TITLE
Increase outbound timeout is not working

### DISCUSSION THREAD

**Discussion by DakshithaDias** [linkerd/linkerd2#13282] [Q&A]
I am trying to increase the global outbound request timeout to 5 seconds. I've applied this change in linkerd-config configmap and restarted all the linkerd pods and also my application pod. Linkerd proxy is auto injected to my app pod.

When i describe the application pod, env variable is correctly set to 5s.
![image](https://github.com/user-attachments/assets/803196c4-b9c8-4bac-adb8-25e1580d953e)

However, when I exec into my application pod and try to curl to a delay pod (I used https://hub.docker.com/r/deminy/delayed-http-response) it doesn't time out at 5 seconds even when I used 10s delay into my curl. Example curl - curl -v 100.65.0.143:80/sleep/10

**Comment by olix0r** [comment]
That timeout controls the TCP connection timeout, which is not an HTTP request timeout. Linkerd doesn't enforce a default HTTP request timeout.

Client-side HTTP request timeouts can be configured with [HTTPRoute resources](https://linkerd.io/2.16/tasks/books/#timeouts), however.
